website/docs/installation/automated-install.md

```diff
@@ -23,3 +23,20 @@ Requires authentik 2023.3
 :::
 
 Set the email address for the default `akadmin` user.
+
+## Kubernetes
+
+Set the `akadmin`user password and token:
```

Comment:
Can you state exactly where they set this? I am assuming in their `.env` file, but let's make it explicit in the instructions. ;-)

Reply:
```suggestion
In the Helm values, set the `akadmin`user password and token:
```